### ISSUE TITLE
bug: Navbar and footer not being rendered on landing page on going back one page

### ISSUE THREAD

**Issue by astro3012** [WalkingPal/walking-pal-web#33]
Clicking on Register and then hitting back on browser results in navbar and footer not being rendered on landing page.

**Comment by ashuvssut** [comment]
@eemike07 see contributing guidelines (dev enviornment setup)

**Comment by eemike07** [comment]
I do see the issue when I navigate to the deployed website: https://walkingpal.in/

However, when deployed locally on my machine, navigating to `/register` results with the following error:  
![image](https://user-images.githubusercontent.com/2515617/193441067-92dc5d4d-b7d3-4119-8863-ce77d75fc640.png)


Am I missing a setup step?  
I ran `yarn install` and `yarn dev` as specified in the contributing guidelines.

**Comment by eemike07** [comment]
I was able to replicate the issue on my machine after removing -
```html
<ReCAPTCHA
ref={recaptchaRef}
size="invisible"
sitekey={process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY as string}
onChange={onReCAPTCHAChange}
/>
```

**Comment by ashuvssut** [comment]
yes. we did not add the steps to add your own recaptcha. you can open an issue for docs changes. 

Its all about going to google's recaptcha sit and doing some stuff there and creating a .local.env files locally

ref https://prateeksurana.me/blog/integrating-recaptcha-with-next/

**Comment by eemike07** [comment]
I was able to resolve the issue.  
See PR for details: https://github.com/WalkingPal/walking-pal-web/pull/44

**Comment by ashuvssut** [comment]
fixed through PR #14 . Hence, Closing the issue
